tests/verify_run.cpp

```diff
@@ -102,7 +102,7 @@ int authenticate_json(std::string json_file, const bool allow_append)
 		return 1;
 	}
 
-	state.compute_mesh_stats();
+	// state.compute_mesh_stats();
```

Comment:
Is this automatically computed now?

Reply:
it is not necessary for the test, no?

Reply:
We use 
```
    out["err_l2"] = state.stats.l2_err;
	out["err_h1"] = state.h1_err;
	out["err_h1"] = state.stats.h1_err;
	out["err_h1_semi"] = state.h1_semi_err;
	out["err_h1_semi"] = state.stats.h1_semi_err;
	out["err_linf"] = state.linf_err;
	out["err_linf"] = state.stats.linf_err;
	out["err_linf_grad"] = state.grad_max_err;
	out["err_linf_grad"] = state.stats.grad_max_err;
	out["err_lp"] = state.lp_err;
	out["err_lp"] = state.stats.lp_err;
```
are these always computed?

Reply:
no, they are computed in `state.compute_errors()`, the other computes the number of pk elements and other mesh stuff

Reply:
https://github.com/polyfem/polyfem/blob/a11c67e9cfc2dcedf53fc737dbe83f43377ae98e/src/polyfem/io/OutData.cpp#L2387